scripts/onnx-dev-setup.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -ex
+shopt -s expand_aliases
+
+# Checking to see if CuDNN is present
+if [ -f /usr/local/cuda/include/cudnn.h ]; then
+  echo "Cudnn header already exists!!"
+
+else 
+  sudo cp -R /home/engshare/third-party2/cudnn/6.0.21/src/cuda/include/* /usr/local/cuda/include/
+  sudo cp -R /home/engshare/third-party2/cudnn/6.0.21/src/cuda/lib64/* /usr/local/cuda/lib64/
+fi
+
+# make sure we find CUDA and that nvcc is runnable
+export PATH=/usr/local/cuda/bin:$PATH
+export LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
+nvcc --version
+
+#Install Caffe2 requirements from the internal yum repo
+sudo yum install python-virtualenv freetype-devel libpng-devel glog gflags protobuf protobuf-devel protobuf-compiler -y
+
+#Installing cmake
+sudo yum remove cmake3 -y
+sudo yum install cmake -y
+sudo yum install autoconf -y
+
+#Proxy setup
+alias with_proxy="HTTPS_PROXY=http://fwdproxy.any:8080 HTTP_PROXY=http://fwdproxy.any:8080 FTP_PROXY=http://fwdproxy.any:8080 https_proxy=http://fwdproxy.any:8080 http_proxy=http://fwdproxy.any:8080 ftp_proxy=http://fwdproxy.any:8080 http_no_proxy='\''*.facebook.com|*.tfbnw.net|*.fb.com'\'"
+
+#Create a virtualenv, activate it, upgrade pip
+cd ~
+with_proxy virtualenv venv
+source venv/bin/activate
+with_proxy pip install pip setuptools -U
+
+#Install other Caffe2 requirements
+rpm -q protobuf # check the version and if necessary update the value below
+# Todo - Add Grep to find protobuf version
+with_proxy pip install future numpy protobuf==2.6.1 ninja pytest-runner
```

Comment:
protobuf python package 2.6.1 doesn't include cpp implementation, should install the latest version.

Reply:
@bddppq so in this case just leave the version blank?

Reply:
@jspisak yep

Since this is doing in a fresh virtualenv, we don't need to consider the case that people have already installed an old version of protobuf (and so pip will just happily skip redoing the installation of protobuf), otherwise we will need to add -U to force it to update protobuf to the newest version.